api/hooks/validate/validators/sin.validator.js

```diff
@@ -0,0 +1,28 @@
+const validateLuhn = string => {
```

Comment:
SIN validation seems like it's not working, as there's no error on `abc`.

<img width="680" alt="Screen Shot 2019-12-19 at 11 52 48" src="https://user-images.githubusercontent.com/950486/71196975-37632e00-2256-11ea-9736-9348cf2b83dc.png">

Reply:
Oh, oops - I took out an early bailout in the SIN validator because i thought the Luhn algorithm would catch this circumstance. Should be good now.